src/antsibull_core/venv.py

```diff
@@ -10,7 +10,7 @@
 import asyncio
 import os
 import sys
-import venv
+import venv  # pyre-ignore[21]
```

Comment:
@gotmax23 any idea why pyre suddenly cannot find `venv` anymore?

Reply:
I have no idea.... I'm somewhat inclined to drop pyre in favor of the more widely adopted mypy instead of using two different type checkers.

Reply:
We did have some cases where pyre found bugs that mypy didn't. So I'd like to keep it as long as it doesn't cause too many problems.

It's hard to say when that boundary is reached though...